### DISCUSSION TITLE
Improve password generator UX

### DISCUSSION THREAD

**Discussion by Olyno** [supabase/supabase#8242] [Feature Requests]
Hi 👋🏻 
I think it could be interesting to improve the random password generation. Few people (including me) pay attention to small texts. I understand the current text as a help more than a text with an action behind. Maybe add a button to generate the password just here could be an idea:

![See here: https://cdn.discordapp.com/attachments/867348150741237790/983867470081310750/unknown.png](https://cdn.discordapp.com/attachments/867348150741237790/983867470081310750/unknown.png)

*Initial suggestion from n10000k on Discord*

**Comment by saltcod** [comment]
Hey @Olyno !   We have this password generator when you create a new project:
https://github.com/supabase/supabase/pull/7249

**Reply by Olyno** [reply]
Nevermind, I then change my suggestion ahah. Would it be possible to improve the user experience on this feature? Few people (including me) pay attention to small texts. I understand this text as a help more than a text with an action behind.

Would it be possible to turn it into a button? Or something equivalent?
